index.html

```diff
@@ -0,0 +1,247 @@
+                </li>
+                <li>
+                    <button class="tag_button">javascript</button>
+                </li>
+            </ul>
+
+            <a href="#" class="button-outline">See Project</a>
+
+        </div>
+        <img class="image is_desktop" src="assets/images/banners/desktop_uber.svg" alt="uber" />
+    </section>
+
+    <section class="about">
+
+        <div class="info">
+            <h1 class="heading__title fw-700">About Myself</h1>
+            <p class="heading__desc">
+                I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
+            <p class="heading__connect">LET’S CONNECT</p>
+            <div class="heading__icons">
+                <ul class="heading__icons__ul ul flex">
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/twitter.svg" alt="twitter icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/linkedin.svg" alt="linkedin icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/medium.svg" alt="github icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/github.svg" alt="github icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/angellist.svg" alt="angellist icon"></a></li>
+                </ul>
+            </div>
+        </div>
+
+        <ul class="casualose ul">
+            <li>
+                <div class="header">
+                    <span>Language</span>
+                    <img src="assets/images/arrow_down.svg" alt="Down Arrow">
+                </div>
+
+                <img class="is_desktop" src="assets/images/languages_desktop.svg" alt="Down Arrow">
+                <img class="is_mobile full_width" src="assets/images/languages_mobile.svg" alt="Down Arrow">
+
+            </li>
+            <li>
+                <div class="header border-bottom">
+                    <span>Frameworks</span>
+                    <img src="assets/images/arrow_right.svg" alt="Right arrow">
+                </div>
+            </li>
+            <li>
+                <div class="header border-bottom">
+                    <span>Skills</span>
+                    <img src="assets/images/arrow_right.svg" alt="Right arrow">
+                </div>
+            </li>
+        </ul>
+
+    </section>
```

Comment:
- [ ] Kindly check the Figma design to confirm how this section was styled on the desktop version of the project. At this moment, your design looks different to what we have on the Figma design. Take a look. :point_down: 

### Your Design
![your-about-desktop](https://user-images.githubusercontent.com/26597432/236410784-dc3b2e72-db4b-4c6c-8a55-8ca381ded645.png)

### Figma Design
![temp-about-desktop](https://user-images.githubusercontent.com/26597432/236410821-b5c8f1a9-b77d-49ed-95f0-449d75fcbf0c.png)